packages/components/src/DateFnsLocale/distanceInWords/distanceInWords.js

```diff
@@ -0,0 +1,96 @@
+function buildDistanceInWords(t) {
+	return {
+		lessThanXSeconds: {
+			one: t('DATE_FNS_LESS_THAN_SECOND', { defaultValue: 'less than a second' }),
+			other: t('DATE_FNS_LESS_THAN_SECOND_plural', { defaultValue: 'less than [count] seconds' }),
```

Comment:
Why not `{{count}}`? And should we not pass `count` to i18next, for special plural cases in other languages?